### PR TITLE
Enhance time formatting with new hide options

### LINKED ISSUE #70: add option to hide seconds: usefull for showing time in months, day, hour, minutes
The package is awesome. But I think there should be an option to hide seconds in the view.
image
As you can see here, it's a time in queue. I want to hide seconds in it. But there is no option to hide this.   Could you please add that option?

### PR DESCRIPTION
**Pull request by gerbermichi**
Added options `hideYears`, `hideYearsAndDays`, and `hideSeconds` to allow for more customizable time formatting.

Fixes #70

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -75,6 +75,27 @@ export type Options = {
 	@default false
 	*/
 	readonly colonNotation?: boolean;
+
+	/**
+	 Hides the years and shows the hidden years additionally as days (365 per year).: `1y 3d 5h 1m 45s` → `368d 5h 1m 45s`.
+
+	 @default false
+	 */
+	readonly hideYears?: boolean;
```
Comment (sindresorhus) [inline index.d.ts:84]: Should be singular, as it should be read as "hide year component".

File: index.d.ts [hunk 2]
```diff
@@ -75,6 +75,27 @@ export type Options = {
 	@default false
 	*/
 	readonly colonNotation?: boolean;
+
+	/**
+	 Hides the years and shows the hidden years additionally as days (365 per year).: `1y 3d 5h 1m 45s` → `368d 5h 1m 45s`.
+
+	 @default false
+	 */
```
Comment (sindresorhus) [inline index.d.ts:83]: Incorrect indentation.

File: test.js [hunk 3]
```diff
@@ -379,6 +379,46 @@ runTests({
...
+});
+
+runTests({
+	title: 'have hideYearsAndDays,hideSeconds and colonNotation options',
+	cases: [
+		[(1000 * 60 * 60 * 15) + (1000 * 60 * 59) + (1000 * 59) + 543, {hideSeconds: true, hideYearsAndDays: true, colonNotation: true}, '15:59'],
+		[(1000 * 60 * 67 * 24 * 465) + (1000 * 60 * 60 * 15) + (1000 * 60 * 59) + (1000 * 59) + 543, {hideSeconds: true, hideYearsAndDays: true, colonNotation: true}, '12477:59'],
+	],
+});
+
```
Comment (sindresorhus) [inline test.js:421]: I would like to see some more tests on how to works together with the other options.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
Thanks for contributing.

**Comment by sindresorhus** [comment]
The readme needs to be updated too.

File: test.js (hunk 3 above)
Comment (gerbermichi) [reply test.js:442]: Hi, I added numerous test cases. Do you have any specific in mind?

**Pull request merged**